R/learn_cvpairs.R

```diff
@@ -1,37 +1,37 @@
 #' Cross-validation pairs
 #'
-#' Add cross validation pairs to a pipelearner object. \code{n} random
-#' partitions will try to be created unless \code{k} is not NULL, in which case
-#' k folds will be created. See \code{\link[modelr]{crossv_mc}} for more
-#' details.
+#' Add cross validation pairs to a pipelearner object. By default will create
+#' a random partition of the data. Can use any function which creates
+#' \code{train} and \code{test} columns of \code{resample} objects and column
+#' of \code{id} integers. See \code{\link[modelr]{crossv_mc}} and
+#' \code{\link[modelr]{resample}} for details.
 #'
 #' Will expect a pipelearner object, but will also accept a data frame for the
 #' parameter \code{pl}. In the case that a data frame is used, it will first
 #' coerce it to a pipelearner object via \code{\link{pipelearner}}.
 #'
 #' @inheritParams pipelearner_params
-#' @inheritParams modelr::crossv_mc
+#' @param .f a function which takes a \code{data} argument
```

Comment:
This needs to be expanded because the output of the function is expected to conform to a very specific structure. Ie, it must take a `data.frame` and return ...